import_owmat.py

```diff
@@ -25,22 +26,32 @@ def cleanUnusedMaterials(materials):
     bpy.context.scene.update()
     return (t, m)
 
+
 def mutate_texture_path(file, new_ext):
     return os.path.dirname(file) + "//" + os.path.splitext(os.path.basename(file))[0] + new_ext
 
+
 def load_textures(texture, root, t):
+    """ Loads an overwatch texture file.
+
+    Will attempt to load the default path (DDS) first,
+    then fall back to Tiff, and lastly TGA.
+    """
     realpath = texture
     if not os.path.isabs(realpath):
         realpath = os.path.normpath('%s/%s' % (root, realpath))
         fn = os.path.splitext(os.path.basename(realpath))[0]
-    if not os.path.exists(realpath):
-        mutate_texture_path(realpath, ".tga")
     tif_file = mutate_texture_path(realpath, ".tif")
     is_tif = False
     if os.path.exists(tif_file):
         realpath = tif_file
         is_tif = True
-    
+
+    tga_file = mutate_texture_path(realpath, ".tga")
+    if not os.path.exists(tif_file) and os.path.exists(tga_file):
+        realpath = tga_file
+        is_tif = False
```

Comment:
I'm not sure what the `is_tif` variable is used for, but it seems to determine the bump/normal map intensity. Can someone explain why that is?

Reply:
That was me being stupid. I thought that normal map strength should be -1 only if the texture is loaded from a TIF. Turns out that it should always be -1.

I have it fixed locally.

Reply:
Not sure if the normal map strength should be -1. Wouldn't that turn off the normal map effect? Why even have it at all then?

Reply:
The normal maps are effectively inverted. Standard Blender supports using -1 as the normal map strength, but it turns out that the Normal Map node doesn't support this.

I'll fix it.